x-pack/test/security_solution_cypress/cypress/objects/rule.ts

```diff
@@ -523,7 +523,63 @@ export const getEditedRule = (): QueryRuleCreateProps =>
     tags: [...(getExistingRule().tags || []), 'edited'],
   });
 
+export const expectedExportedRules = (responses: Array<Cypress.Response<RuleResponse>>): string => {
```

Comment:
Why do we duplicate the code of the `expectedExportedRule` function?

Reply:
This diff looks kind of weird but it's abstracting it out into 2 functions and just keeping the shared logic as a separate helper. Could probably just be changed into one `expectedExportedRules` function